src/components/UserDetails/UserDetails.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import {
+  Box,
+  Link,
+  makeStyles,
+  TextareaAutosize,
+  Typography,
+} from '@material-ui/core';
+import Avatar from 'components/Avatar';
+import { isNewbie } from 'utils';
+import { UserRole } from 'buddy-app-schema';
+import { UserDetailsProps } from './types';
+
+const useStyles = makeStyles(theme => ({
+  notesTextarea: {
+    width: '15rem',
+    minHeight: '10rem',
+    borderWidth: '1px',
+    borderColor: theme.palette.primary.dark,
+  },
+  wrapper: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginTop: theme.spacing(2),
+  },
+  avatar: {
+    width: '10rem',
+  },
+  label: {
+    fontWeight: theme.typography.fontWeightBold,
+    marginTop: theme.spacing(0.5),
+  },
+  detailText: {
+    minHeight: '1.8rem',
+  },
+}));
+
+const UserDetails: React.FC<UserDetailsProps> = props => {
+  const { wrapper, avatar, notesTextarea, label, detailText } = useStyles();
+  const {
+    photo,
+    name,
+    position,
+    startDate,
+    email,
+    phoneNumber,
+    notes,
+    role,
+  } = props.details;
+
+  const isNotesVisible = (role: UserRole) => isNewbie(role);
+
+  return (
+    <div data-testid='contact-details-page'>
+      <Box className={wrapper}>
```

Comment:
You can remove this div and pass ` data-testid='contact-details-page' to Box component. And minimize DOM tree

Reply:
Done. 3176e1bb0d0a5f5a1a039bd0a390a5ec91925c79